client/lib/cart-values/cart-items.js

```diff
@@ -465,15 +465,15 @@ export function getGoogleApps( cart ) {
 /**
  * Creates a new shopping cart item for G Suite or Google Workspace.
  *
- * @param {object} properties - list of properties
+ * @param {{domain: string, meta?: string, new_quantity?: number, productSlug: string, quantity?: number, users: [] }} properties - list of properties
```

Comment:
I think you can get `users` with `GSuiteProductUser` but because this is jsdoc you may have to do a little awkward juggling. I think this will work (untested):

```
/**
  * @typedef {import('@automattic/shopping-cart').GSuiteProductUser} GSuiteProductUser
  * @param {{domain: string, meta?: string, new_quantity?: int, productSlug: string, quantity?: int, users: GSuiteProductUser[] }} properties - list of properties
```

Reply:
Thanks, I'll update it! That seems to work in WebStorm (I can navigate to the type declaration) but I'm not sure if there is a better way to test it though.

Reply:
Good enough for now! Thanks! I may convert this file to TS soon anyway so we don't need to have it all perfect.